RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFitOOTPileupCorrection.h

```diff
@@ -130,6 +130,7 @@ class PulseShapeFitOOTPileupCorrection
     double pedSig_;
     double noise_;    
     HcalTimeSlew::BiasSetting slewFlavor_;    
+    const double overlapLimit = 1.0;
```

Comment:
This is ns right?  I'm not sure this is big enough, since each data point in the fit is 25 ns wide.  Hence pulses could be up to 24.9999 ns apart and still be referring to the same bin on the distribution.  In my opinion it would be better to check which bin the pulses refer to.  For the short term though I think this code is okay although I'll make the overlapLimit user configurable (I can do this in another pull request).